common/enablers/enableEpochsHandler_test.go

```diff
@@ -217,26 +220,37 @@ func TestNewEnableEpochsHandler_EpochConfirmed(t *testing.T) {
 		assert.True(t, handler.IsAlwaysSaveTokenMetaDataEnabled())
 		assert.True(t, handler.IsRuntimeCodeSizeFixEnabled())
 		assert.True(t, handler.IsConsistentTokensValuesLengthCheckEnabled())
-		assert.False(t, handler.IsKeepExecOrderOnCreatedSCRsEnabled())
-		assert.False(t, handler.IsMultiClaimOnDelegationEnabled())
-		assert.False(t, handler.IsChangeUsernameEnabled())
+		assert.True(t, handler.IsKeepExecOrderOnCreatedSCRsEnabled())
+		assert.True(t, handler.IsMultiClaimOnDelegationEnabled())
+		assert.True(t, handler.IsChangeUsernameEnabled())
+		assert.True(t, handler.IsSetSenderInEeiOutputTransferFlagEnabled())
+		assert.True(t, handler.IsFixAsyncCallbackCheckFlagEnabled())
+		assert.True(t, handler.IsSaveToSystemAccountFlagEnabled())
+		assert.True(t, handler.IsCheckFrozenCollectionFlagEnabled())
+		assert.True(t, handler.IsSendAlwaysFlagEnabled())
+		assert.True(t, handler.IsValueLengthCheckFlagEnabled())
+		assert.True(t, handler.IsCheckTransferFlagEnabled())
+		assert.True(t, handler.IsTransferToMetaFlagEnabled())
+		assert.True(t, handler.IsESDTNFTImprovementV1FlagEnabled())
+		assert.True(t, handler.IsRefactorPeersMiniBlocksFlagEnabled())
+		assert.True(t, handler.IsSetGuardianEnabled())
+
 	})
 	t.Run("flags with == condition should be set, along with all >=", func(t *testing.T) {
 		t.Parallel()
 
-		epoch := uint32(80)
+		epoch := uint32(85)
```

Comment:
when we will merge the master -> rc/v1.5.0 -> rc/v1.6.0 this line will become `epoch := uint32(math.MaxUint32)`.